pkg/lib/v0_2_0/events.go

```diff
@@ -7,17 +7,54 @@ import (
 	"fmt"
 	"github.com/cloudevents/sdk-go/v2/protocol"
 	"github.com/keptn/go-utils/pkg/api/models"
+	"github.com/keptn/go-utils/pkg/lib/keptn"
 	"log"
 	"time"
 
-	"github.com/keptn/go-utils/pkg/lib/keptn"
-
 	cloudevents "github.com/cloudevents/sdk-go/v2"
 	httpprotocol "github.com/cloudevents/sdk-go/v2/protocol/http"
 )
 
 const MAX_SEND_RETRIES = 3
 
+// CloudEventsHTTPEventSender sends CloudEvents via HTTP
+type CloudEventsHTTPEventSender struct {
```

Comment:
Ideally, the `ClooudEventsHTTPEventSender` has a dependendy to `coudevents.Client` injected. This way we could swap it by a mock and unit test the event sender nicely.